app/download/views.py

```diff
@@ -51,7 +51,7 @@ def download_document_b64(service_id, document_id):
         abort(404)
 
     filename = request.args.get('filename')
-    sending_method = request.args.get('sending_method')
+    sending_method = request.args.get('sending_method', 'link')
```

Comment:
`None` is treated as `link`. Making it explicit here allows the url returned by `url_for(
        'download.download_document_b64',...)` to be simpler.